AjaxControlToolkit/AjaxFileUpload/AjaxFileUpload.cs

```diff
@@ -389,7 +389,8 @@ public AjaxFileUpload()
         }
 
         public static string BuildRootTempFolder() {
-            var rootTempFolder = Path.Combine(AjaxFileUploadHelper.RootTempFolderPath, TemporaryUploadFolderName);
+            var rootTempFolder = AjaxFileUploadHelper.RootTempFolderPath ?? Path.Combine(Path.GetTempPath(), TemporaryUploadFolderName);
+
             if(!Directory.Exists(rootTempFolder))
```

Comment:
This attempts to create `AjaxFileUploadHelper.RootTempFolderPath`. We decided not to do that. Instead, throw if it doesn't exist.